app/models/course.rb

```diff
@@ -159,4 +159,21 @@ def funding
       'fee'
     end
   end
+
+  def publish_sites
+    site_statuses.status_new_status.each(&:status_running!)
+    site_statuses.status_running.unpublished_on_ucas.each(&:published_on_ucas!)
+
+    # as update_all is called and doesn't trigger a save this has to be manually tested
+    update(changed_at: Time.now.utc)
+  end
+
+  def publish_enrichment(current_user)
+    enrichments.draft
+    .update(status: 'published',
```

Comment:
Could this be moved to an instance method on `Enrichment` called `#published` or something similar?

Reply:
good idea. done